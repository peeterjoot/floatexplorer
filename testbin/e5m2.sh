./floatexplorer --e5m2 0x00
./floatexplorer --e5m2 0x01
./floatexplorer --e5m2 0x02
./floatexplorer --e5m2 0x03
./floatexplorer --e5m2 0x04
./floatexplorer --e5m2 0x05
./floatexplorer --e5m2 0x06
./floatexplorer --e5m2 0x07
./floatexplorer --e5m2 0x08
./floatexplorer --e5m2 0x09
./floatexplorer --e5m2 0x0A
./floatexplorer --e5m2 0x0B
./floatexplorer --e5m2 0x0C
./floatexplorer --e5m2 0x0D
./floatexplorer --e5m2 0x0E
./floatexplorer --e5m2 0x0F
./floatexplorer --e5m2 0x10
./floatexplorer --e5m2 0x11
./floatexplorer --e5m2 0x12
./floatexplorer --e5m2 0x13
./floatexplorer --e5m2 0x14
./floatexplorer --e5m2 0x15
./floatexplorer --e5m2 0x16
./floatexplorer --e5m2 0x17
./floatexplorer --e5m2 0x18
./floatexplorer --e5m2 0x19
./floatexplorer --e5m2 0x1A
./floatexplorer --e5m2 0x1B
./floatexplorer --e5m2 0x1C
./floatexplorer --e5m2 0x1D
./floatexplorer --e5m2 0x1E
./floatexplorer --e5m2 0x1F
./floatexplorer --e5m2 0x20
./floatexplorer --e5m2 0x21
./floatexplorer --e5m2 0x22
./floatexplorer --e5m2 0x23
./floatexplorer --e5m2 0x24
./floatexplorer --e5m2 0x25
./floatexplorer --e5m2 0x26
./floatexplorer --e5m2 0x27
./floatexplorer --e5m2 0x28
./floatexplorer --e5m2 0x29
./floatexplorer --e5m2 0x2A
./floatexplorer --e5m2 0x2B
./floatexplorer --e5m2 0x2C
./floatexplorer --e5m2 0x2D
./floatexplorer --e5m2 0x2E
./floatexplorer --e5m2 0x2F
./floatexplorer --e5m2 0x30
./floatexplorer --e5m2 0x31
./floatexplorer --e5m2 0x32
./floatexplorer --e5m2 0x33
./floatexplorer --e5m2 0x34
./floatexplorer --e5m2 0x35
./floatexplorer --e5m2 0x36
./floatexplorer --e5m2 0x37
./floatexplorer --e5m2 0x38
./floatexplorer --e5m2 0x39
./floatexplorer --e5m2 0x3A
./floatexplorer --e5m2 0x3B
./floatexplorer --e5m2 0x3C
./floatexplorer --e5m2 0x3D
./floatexplorer --e5m2 0x3E
./floatexplorer --e5m2 0x3F
./floatexplorer --e5m2 0x40
./floatexplorer --e5m2 0x41
./floatexplorer --e5m2 0x42
./floatexplorer --e5m2 0x43
./floatexplorer --e5m2 0x44
./floatexplorer --e5m2 0x45
./floatexplorer --e5m2 0x46
./floatexplorer --e5m2 0x47
./floatexplorer --e5m2 0x48
./floatexplorer --e5m2 0x49
./floatexplorer --e5m2 0x4A
./floatexplorer --e5m2 0x4B
./floatexplorer --e5m2 0x4C
./floatexplorer --e5m2 0x4D
./floatexplorer --e5m2 0x4E
./floatexplorer --e5m2 0x4F
./floatexplorer --e5m2 0x50
./floatexplorer --e5m2 0x51
./floatexplorer --e5m2 0x52
./floatexplorer --e5m2 0x53
./floatexplorer --e5m2 0x54
./floatexplorer --e5m2 0x55
./floatexplorer --e5m2 0x56
./floatexplorer --e5m2 0x57
./floatexplorer --e5m2 0x58
./floatexplorer --e5m2 0x59
./floatexplorer --e5m2 0x5A
./floatexplorer --e5m2 0x5B
./floatexplorer --e5m2 0x5C
./floatexplorer --e5m2 0x5D
./floatexplorer --e5m2 0x5E
./floatexplorer --e5m2 0x5F
./floatexplorer --e5m2 0x60
./floatexplorer --e5m2 0x61
./floatexplorer --e5m2 0x62
./floatexplorer --e5m2 0x63
./floatexplorer --e5m2 0x64
./floatexplorer --e5m2 0x65
./floatexplorer --e5m2 0x66
./floatexplorer --e5m2 0x67
./floatexplorer --e5m2 0x68
./floatexplorer --e5m2 0x69
./floatexplorer --e5m2 0x6A
./floatexplorer --e5m2 0x6B
./floatexplorer --e5m2 0x6C
./floatexplorer --e5m2 0x6D
./floatexplorer --e5m2 0x6E
./floatexplorer --e5m2 0x6F
./floatexplorer --e5m2 0x70
./floatexplorer --e5m2 0x71
./floatexplorer --e5m2 0x72
./floatexplorer --e5m2 0x73
./floatexplorer --e5m2 0x74
./floatexplorer --e5m2 0x75
./floatexplorer --e5m2 0x76
./floatexplorer --e5m2 0x77
./floatexplorer --e5m2 0x78
./floatexplorer --e5m2 0x79
./floatexplorer --e5m2 0x7A
./floatexplorer --e5m2 0x7B
./floatexplorer --e5m2 0x7C
./floatexplorer --e5m2 0x7D
./floatexplorer --e5m2 0x7E
./floatexplorer --e5m2 0x7F
./floatexplorer --e5m2 0x80
./floatexplorer --e5m2 0x81
./floatexplorer --e5m2 0x82
./floatexplorer --e5m2 0x83
./floatexplorer --e5m2 0x84
./floatexplorer --e5m2 0x85
./floatexplorer --e5m2 0x86
./floatexplorer --e5m2 0x87
./floatexplorer --e5m2 0x88
./floatexplorer --e5m2 0x89
./floatexplorer --e5m2 0x8A
./floatexplorer --e5m2 0x8B
./floatexplorer --e5m2 0x8C
./floatexplorer --e5m2 0x8D
./floatexplorer --e5m2 0x8E
./floatexplorer --e5m2 0x8F
./floatexplorer --e5m2 0x90
./floatexplorer --e5m2 0x91
./floatexplorer --e5m2 0x92
./floatexplorer --e5m2 0x93
./floatexplorer --e5m2 0x94
./floatexplorer --e5m2 0x95
./floatexplorer --e5m2 0x96
./floatexplorer --e5m2 0x97
./floatexplorer --e5m2 0x98
./floatexplorer --e5m2 0x99
./floatexplorer --e5m2 0x9A
./floatexplorer --e5m2 0x9B
./floatexplorer --e5m2 0x9C
./floatexplorer --e5m2 0x9D
./floatexplorer --e5m2 0x9E
./floatexplorer --e5m2 0x9F
./floatexplorer --e5m2 0xA0
./floatexplorer --e5m2 0xA1
./floatexplorer --e5m2 0xA2
./floatexplorer --e5m2 0xA3
./floatexplorer --e5m2 0xA4
./floatexplorer --e5m2 0xA5
./floatexplorer --e5m2 0xA6
./floatexplorer --e5m2 0xA7
./floatexplorer --e5m2 0xA8
./floatexplorer --e5m2 0xA9
./floatexplorer --e5m2 0xAA
./floatexplorer --e5m2 0xAB
./floatexplorer --e5m2 0xAC
./floatexplorer --e5m2 0xAD
./floatexplorer --e5m2 0xAE
./floatexplorer --e5m2 0xAF
./floatexplorer --e5m2 0xB0
./floatexplorer --e5m2 0xB1
./floatexplorer --e5m2 0xB2
./floatexplorer --e5m2 0xB3
./floatexplorer --e5m2 0xB4
./floatexplorer --e5m2 0xB5
./floatexplorer --e5m2 0xB6
./floatexplorer --e5m2 0xB7
./floatexplorer --e5m2 0xB8
./floatexplorer --e5m2 0xB9
./floatexplorer --e5m2 0xBA
./floatexplorer --e5m2 0xBB
./floatexplorer --e5m2 0xBC
./floatexplorer --e5m2 0xBD
./floatexplorer --e5m2 0xBE
./floatexplorer --e5m2 0xBF
./floatexplorer --e5m2 0xC0
./floatexplorer --e5m2 0xC1
./floatexplorer --e5m2 0xC2
./floatexplorer --e5m2 0xC3
./floatexplorer --e5m2 0xC4
./floatexplorer --e5m2 0xC5
./floatexplorer --e5m2 0xC6
./floatexplorer --e5m2 0xC7
./floatexplorer --e5m2 0xC8
./floatexplorer --e5m2 0xC9
./floatexplorer --e5m2 0xCA
./floatexplorer --e5m2 0xCB
./floatexplorer --e5m2 0xCC
./floatexplorer --e5m2 0xCD
./floatexplorer --e5m2 0xCE
./floatexplorer --e5m2 0xCF
./floatexplorer --e5m2 0xD0
./floatexplorer --e5m2 0xD1
./floatexplorer --e5m2 0xD2
./floatexplorer --e5m2 0xD3
./floatexplorer --e5m2 0xD4
./floatexplorer --e5m2 0xD5
./floatexplorer --e5m2 0xD6
./floatexplorer --e5m2 0xD7
./floatexplorer --e5m2 0xD8
./floatexplorer --e5m2 0xD9
./floatexplorer --e5m2 0xDA
./floatexplorer --e5m2 0xDB
./floatexplorer --e5m2 0xDC
./floatexplorer --e5m2 0xDD
./floatexplorer --e5m2 0xDE
./floatexplorer --e5m2 0xDF
./floatexplorer --e5m2 0xE0
./floatexplorer --e5m2 0xE1
./floatexplorer --e5m2 0xE2
./floatexplorer --e5m2 0xE3
./floatexplorer --e5m2 0xE4
./floatexplorer --e5m2 0xE5
./floatexplorer --e5m2 0xE6
./floatexplorer --e5m2 0xE7
./floatexplorer --e5m2 0xE8
./floatexplorer --e5m2 0xE9
./floatexplorer --e5m2 0xEA
./floatexplorer --e5m2 0xEB
./floatexplorer --e5m2 0xEC
./floatexplorer --e5m2 0xED
./floatexplorer --e5m2 0xEE
./floatexplorer --e5m2 0xEF
./floatexplorer --e5m2 0xF0
./floatexplorer --e5m2 0xF1
./floatexplorer --e5m2 0xF2
./floatexplorer --e5m2 0xF3
./floatexplorer --e5m2 0xF4
./floatexplorer --e5m2 0xF5
./floatexplorer --e5m2 0xF6
./floatexplorer --e5m2 0xF7
./floatexplorer --e5m2 0xF8
./floatexplorer --e5m2 0xF9
./floatexplorer --e5m2 0xFA
./floatexplorer --e5m2 0xFB
./floatexplorer --e5m2 0xFC
#./floatexplorer --e5m2 0xFD
#./floatexplorer --e5m2 0xFE
#./floatexplorer --e5m2 0xFF
