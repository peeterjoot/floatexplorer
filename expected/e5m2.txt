value:    3
hex:      42
bits:     01000010
sign:     0
exponent:  10000                        (15 +1)
mantissa:       10
number:       1.10 x 2^(1)

